File: src/main/java/com/example/eyeofsauron/repository/MarkerReaderRepository.java
package com.example.eyeofsauron.repository;

import com.example.eyeofsauron.entity.MarkerReader;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

@Repository
public interface MarkerReaderRepository extends JpaRepository<MarkerReader, Long>, JpaSpecificationExecutor<MarkerReader> {
}

File: src/main/kotlin/com/example/eyeofsauron/repository/MarkerReaderRepository.kt
package com.example.eyeofsauron.repository

import com.example.eyeofsauron.entity.MarkerReader
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.stereotype.Repository

@Repository
interface MarkerReaderRepository : JpaRepository<MarkerReader, Long>, JpaSpecificationExecutor<MarkerReader>